packages/extension-api/src/extension-api.d.ts

```diff
@@ -1868,6 +1868,7 @@ declare module '@podman-desktop/api' {
     Labels: { [label: string]: string };
     Containers: number;
     History?: string[];
+    Digest?: string;
```

Comment:
I would expect to always have the digest, using docker or podman engine

as in case of docker it would be `sha256:ID`

Reply:
Updated

Reply:
it is still marked optional there ?